xlb/operator/boundary_condition/bc_regularized.py

```diff
@@ -222,6 +218,15 @@ def functional_velocity(
             # Find normal vector
             normals = get_normal_vectors(missing_mask)
 
+            # Find the value of u from the missing directions
+            for l in range(wp.static(_q)):
```

Comment:
I don't think we need wp.static here

Reply:
wp.static() only makes a difference if you have an expression instead of a simple constant.